test/test-pages/gcba/sdk-controller.ts

```diff
@@ -0,0 +1,307 @@
+    const parentNode = document.getElementById('sessions-element');
+    parentNode.innerHTML = `<table class="table">
+      <thead>
+        <tr>
+          <th scope="col" scope="row">conversationId</th>
+          <th scope="col">sessionId</th>
+          <th scope="col">session state</th>
+        </tr>
+      </thead>
+      <tbody id="${tableBodyId}">
+      </tbody>
+    </table>`;
+    tableBody = document.getElementById(tableBodyId);
+  }
+
+  Object.values(conversationUpdatesToRender).forEach(session => {
+    const isSessionActive = session ? session.state === 'active' : undefined;
+
+    html += `<tr>
+    <th scope="row">${session.conversationId}</th>
+    <td>${session.id}</td>
+    <td class="${isSessionActive ? 'text-success' : 'text-danger'}">
+      ${session.state}
+    </td>
+  </tr>`
+  });
+  tableBody.innerHTML = html;
+}
+
+function cancelPendingSession (params) {
+  let output = `${_getLogHeader('cancelPendingSession')}
+    sessionId: ${params.sessionId}
+    conversationId: ${params.conversationId}`;
+
+  pendingSessions = pendingSessions.filter(s => s.conversationId !== params.conversationId);
+  renderPendingSessions();
+  writeToLog(output);
+}
+
+function handledPendingSession (params) {
+  let output = `${_getLogHeader('handledPendingSession')}
+    sessionId: ${params.sessionId}
+    conversationId: ${params.conversationId}`;
+
+  pendingSessions = pendingSessions.filter(s => s.conversationId !== params.conversationId);
+  renderPendingSessions();
+  writeToLog(output);
+}
+
+async function sessionStarted (session: IExtendedMediaSession) {
+  let output = `${_getLogHeader('sessionStarted')}
+    conversationId: ${session.conversationId}
+    sessionId: ${session.sid}`;
+  conversationUpdatesToRender[session.conversationId] = session;
+
+  session.on('connectionState', () => renderSessions());
+  renderSessions();
+  writeToLog(output);
+
+  const screenStream = await webrtcSdk.media.startDisplayMedia();
+
+  // create metadatas
+  const track = screenStream.getTracks()[0];
+  const { height, width, deviceId } = track.getSettings();
+  const screenRecordingMetadatas = [
+    {
+      trackId: track.id,
+      screenId: deviceId, // some applications give you a deviceId on the track which is uniquely tied to a specific monitor
+      originX: 0,
+      originY: 0,
+      resolutionX: width,
+      resolutionY: height,
+    primary: true,
+    }
+  ];
+
+  webrtcSdk.acceptSession({ conversationId: session.conversationId, sessionType: session.sessionType, mediaStream: screenStream, screenRecordingMetadatas });
+}
+
+function trace (level, message, details) {
+  let output = `${_getLogHeader('trace')}\n`;
+  output += `  level: ${level}\n  message: ${message}\n  details: ${details}`;
+
+  const logTraces = (document.getElementById('log-traces-check') as any).checked;
+  if (logTraces) {
+    writeToLog(output);
+  }
+}
+
+function error (error: SdkError) {
+  let output = `${_getLogHeader('error')}
```

Comment:
Codacy has a fix for the issue: ['output' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9580329)
```suggestion
  const output = `${_getLogHeader('error')}
```